packages/cli/src/commands/generate.js

```diff
@@ -80,6 +80,9 @@ export default {
 
     // Set flag to keep the prerendering behaviour
     config._legacyGenerate = true
+    if (config.build) {
+      config.build.parallel = false
```

Comment:
- It would be nice if we can investigate more what is root cause and possibly report/fix in upstream
- It seems a webpack bug (?) but only happening for generate?
- Linking related issue to workaround at least would be nice :)

Reply:
Issue only happens when generate includes build step before, like also #7855. Skipping build in full static solves issue.

Reply:
But only one instance of nuxt initializes webpack right?

Reply:
Update: Added comments for fast merge still think worth checking rootCause instead of patch in longer term :)